ansible_collections/arista/avd/roles/eos_designs/templates/core_interfaces/logic.j2

```diff
@@ -24,20 +25,20 @@
 {%         set node_index = nodes.index(inventory_hostname) %}
 {%         set peer_index = 1 - node_index %}
 {%         set ip_pool_name = p2p_link.ip_pool | arista.avd.default(
-                              link_info.p2p_profile.ip_pool) %}
-
+                                  link_info.p2p_profile.ip_pool, none) %}
 {%         if ip_pool_name is arista.avd.defined and core_interfaces.p2p_links_ip_pools is arista.avd.defined %}
 {%             for pool in core_interfaces.p2p_links_ip_pools %}
 {%                 if pool.name is arista.avd.defined(ip_pool_name) %}
 {%                     set link_info.ip_pool = pool.ipv4_pool | arista.avd.default() %}
+{%                     set link_info.ip_pool_type = pool.type | arista.avd.default("ipv4") %}
 {%                     break %}
 {%                 endif %}
 {%             endfor %}
 {%         endif %}
 {%         set subnet = p2p_link.subnet | arista.avd.default(
-                        link_info.p2p_profile.subnet) %}
+                            link_info.p2p_profile.subnet, none) %}
 {%         set ip_list = p2p_link.ip | arista.avd.default(
-                         link_info.p2p_profile.ip) %}
+                            link_info.p2p_profile.ip, none) %}
```

Comment:
none/null/None is already the default value returned if the default var is not defined.

Reply:
Cool, i removed it again.